Framework/Algorithms/src/EQSANSResolution.cpp

```diff
@@ -2,6 +2,7 @@
 // Includes
 //----------------------------------------------------------------------
 #include "MantidAlgorithms/EQSANSResolution.h"
+#include <math.h>
```

Comment:
See for example https://stackoverflow.com/questions/10694255/c-cmath-vs-math-h-and-similar-c-prefixed-vs-h-extension-headers
Then all math functions should be defined in the `std` namespace. So replace `pow` with `std::pow`